src/test/java/com/parkingapp/parkingservice/infrastructure/entrypoint/rest/ParkingControllerContractTest.java

```diff
@@ -262,6 +262,31 @@ private MockMvcResponse whenARequestToGetParkingByIdIsReceived() {
     @Nested
     class CheckParking {
 
+        @Test
+        void shouldReturnParkingAndStatus() {
```

Comment:
2 tests that include body asserts:

void shouldReturnParkingAndStatusWithParkingDetail() {

void shouldReturnParkingAndStatusWithNoParkingDetail() {

Reply:
done! 👍🏻